packages/web-twig/src/Resources/components/Grid/Grid.stories.twig

```diff
@@ -10,6 +10,22 @@
         {% include '@components/Grid/stories/GridResponsiveColumns.twig' %}
     </DocsSection>
 
+    <DocsSection title="Custom Spacing">
+        {% include '@components/Grid/stories/GridCustomSpacing.twig' %}
+    </DocsSection>
+
+    <DocsSection title="Responsive Custom Spacing">
+        {% include '@components/Grid/stories/GridResponsiveCustomSpacing.twig' %}
+    </DocsSection>
+
+    <DocsSection title="Responsive Custom Vertical Spacing">
+        {% include '@components/Grid/stories/GridResponsiveCustomVerticalSpacing.twig' %}
+    </DocsSection>
+
+    <DocsSection title="Responsive Custom Horizontal Spacing">
+        {% include '@components/Grid/stories/GridResponsiveCustomHorizontalSpacing.twig' %}
+    </DocsSection>
```

Comment:
Please swap the horizontal and vertical examples so anything horizontal always goes first.